CONTRIBUTING.md

```diff
@@ -20,9 +20,40 @@ You can open a new issue with this [issue form](https://github.com/novuhq/novu/i
 
 ### Requirements
 
-- Node.js version v16.15.1
-- MongoDB
-- Redis. To install Redis on your O.S, please follow the below guides
+- Node.js v16.14.0 
+  - To install Node.js v16.14.0 through NVM (Node Version Manager), follow these steps:
+    1. Open your terminal.
+
+    2. Install NVM if you haven't already. You can install NVM by following the instructions at [NVM GitHub](https://github.com/nvm-sh/nvm).
+
+    3. Once NVM is installed, run the following command to install Node.js v16.14.0:
+       
+
+        ```bash
```

Comment:
Can you merge all of these into a single code block?

for line 37 just do node -v # output: v16.x.x

then you can get rid of line 41

Reply:
okay

Reply:
> Can you merge all of these into a single code block?
> 
> for line 37 just do node -v # output: v16.x.x
> 
> then you can get rid of line 41

is it correct now?